.mailmap

```diff
@@ -0,0 +1,2 @@
+Jörn Zaefferer <joern.zaefferer@gmail.com>
+Richard D. Worth <rdworth@gmail.com>
```

Comment:
@rxaviers I just figured out there is a [rule for line feed in JCSC](http://jscs.info/rule/requireLineFeedAtFileEnd), if you allow i can report a new issue and add that rule to globalize's `.jscsrc`.